avocado/core/messages.py

```diff
@@ -335,6 +336,15 @@ def handle(self, message, task, job):
             )
         self._save_to_default_file(message, task)
 
+        # For messages from specific loggers, we preserve their names
+        # and levels so that they are handled appropriately based on
+        # the Avocado job logging configuration
+        log_name = message.get("log_name")
```

Comment:
You are adding new properties to `LogMessage` so you have to update the docstring, because it will be propagated to our documentation.